code/modules/religion/sects/shadow_sect.dm

```diff
@@ -319,93 +325,80 @@
 	ritual_length = 60 SECONDS
 	ritual_invocations = list(
 		"Join us in this darkness ...",
 		"... Protect us from the light ...",
 		"... Destroy those who defy us ...")
 	invoke_msg = "... Let the darkness come and fight!!"
-	favor_cost = 50000
+	favor_cost = 25000
 
 /datum/religion_rites/final_darkness/invoke_effect(mob/living/user, atom/religious_tool)
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
 	if(sect.faithful_used)
 		to_chat(user,"<span class='warning'>This rite has already been used, your favor has been refuned.</span>")
-		GLOB.religious_sect?.adjust_favor(50000, user)
+		GLOB.religious_sect?.adjust_favor(favor_cost, user)
 		return ..()
+	if(user.mind.is_murderbone())
+		sect.Murder = TRUE
 	sect.faithful_used = TRUE
 	var/altar_turf = get_turf(religious_tool)
 	var/obj/structure/destructible/religion/shadow_obelisk/lisk = new(altar_turf)
 	sect.obelisks += lisk
 	lisk.AddComponent(/datum/component/dark_favor, user)
 	lisk.set_light(sect.light_reach, sect.light_power, DARKNESS_INVERSE_COLOR)
 	playsound(altar_turf, 'sound/magic/fireball.ogg', 50, TRUE)
-	priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
-	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate)), 100)
+	if(sect.Murder)
+		priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	else
+		priority_announce("May our lord, [GLOB.deity], bless all the shadows as darkness heals those who worship the night.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate)), 250)
 
 /obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate()
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
 	for(var/obj/structure/destructible/religion/shadow_obelisk/obs in sect.obelisks)
 		START_PROCESSING(SSobj, obs)
-		var/obelisk_turf = get_turf(obs)
-		for(var/i in 1 to 3)
-			var/mob/living/simple_animal/hostile/faithless/faithful/faithful = new(obelisk_turf)
-			faithful.AddComponent(/datum/component/dark_favor, faithful)
-			faithful.set_light(2, -2, DARKNESS_INVERSE_COLOR)
-		playsound(obs, 'sound/hallucinations/wail.ogg', 50, TRUE)
+		if(sect.Murder)
+			var/obelisk_turf = get_turf(obs)
+			if(prob(30))
+				for(var/i in 1 to 3)
+					var/mob/living/simple_animal/hostile/faithless/faithful/faithful = new(obelisk_turf)
+					faithful.AddComponent(/datum/component/dark_favor, faithful)
+					faithful.set_light(2, -2, DARKNESS_INVERSE_COLOR)
+			playsound(obs, 'sound/hallucinations/wail.ogg', 50, TRUE)
+		else
+			playsound(obs, 'sound/magic/fireball.ogg', 50, TRUE)
```

Comment:
I think you should also consider making the ``3`` in ``for(var/i in 1 to 3)`` into a variable or define. A variable is slightly better for testing & admin bus
```suggestion
		if(!sect.Murder)
		    playsound(obs, 'sound/magic/fireball.ogg', 50, TRUE)
		    return
		var/obelisk_turf = get_turf(obs)
		if(prob(30))
			for(var/i in 1 to 3)
				var/mob/living/simple_animal/hostile/faithless/faithful/faithful = new(obelisk_turf)
				faithful.AddComponent(/datum/component/dark_favor, faithful)
				faithful.set_light(2, -2, DARKNESS_INVERSE_COLOR)
		playsound(obs, 'sound/hallucinations/wail.ogg', 50, TRUE)
```